### PR TITLE
Brocade VDX switch integration as part of switch discovery

### PR DESCRIPTION
**Pull request by dpasupathi**
Changes related to on-http repo PR:  Brocade VDX switch integration as part of switch discovery #331 

Added default graph definition to catalog brocade switch.

### CODE REVIEW

**Comment by benbp** [comment]
:+1:

**Comment by lanchongyizu** [comment]
Can one of the admins verify this patch?

File: lib/graphs/brocade-switch-discovery-graph.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        {
+            "label": "catalog-brocade-switch",
+            "taskDefinition": {
+                "friendlyName": "Catalog Brocade Switch",
+                "injectableName": "Task.Inline.Catalog.Switch.Brocade",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/brocade-catalog-version.py",
```
Comment (anhou) [inline lib/graphs/brocade-switch-discovery-graph.js]: @dpasupathi  downloadUrl will break when migrated to API 2.0? could it be decoupled with API version?  @brianparry @RackHD/corecommitters

**Comment by dpasupathi** [comment]
@anhou: I followed the examples present for other network switch graphs. Not sure how to make the API version decoupled. Can you please point me how to do that?

**Comment by anhou** [comment]
@dpasupathi  one reference is  https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/install-centos.js#L14   so maybe it could be  "downloadUrl": "{{api.templates}}/brocade-catalog-version.py"  so it works, could you also help update other two switch files. $ grep -rn downloadUrl on-taskgraph/
on-taskgraph/lib/graphs/arista-switch-discovery-graph.js:18:                            "downloadUrl": "/api/1.1/templates/arista-catalog-version.py",
on-taskgraph/lib/graphs/cisco-switch-discovery-graph.js:18:                            "downloadUrl": "/api/1.1/templates/cisco-catalog-version.py",

File: lib/graphs/brocade-switch-discovery-graph.js (hunk 1 above)
Comment (benbp) [inline lib/graphs/brocade-switch-discovery-graph.js]: @anhou we're going to have this problem with a lot of task definitions, but the 1.1 API should still be operable for long enough to do a mass migration, no?

Comment (brianparry) [inline lib/graphs/brocade-switch-discovery-graph.js]: Sorry, missed this conversation.  Yes, @benbp, the plan is to do a mass migration. @keedya is actually starting to do some work on this now.

**Comment by brianparry** [comment]
ok to test

**Comment by brianparry** [comment]
jenkins, test this please

**Comment by brianparry** [comment]
@anhou Do you have any objection to merging this change?

**Comment by anhou** [comment]
:+1:

**Pull request merged**